### DISCUSSION TITLE
How works `../` in `vercel build` command

### DISCUSSION THREAD

**Discussion by dtokarczyk** [vercel/vercel#8948] [Help] [answered]
I cannot understand how works and how a deploy custom relative path in `vercel build`.

If I use `../` I go back two levels up.

See my CLI commands where I was trying to execute `vercel build` in `~/Development/company/project`

```
user@MacBook-Pro-Damian web % pwd
/Users/user/Development/company/project/packages/website/web
user@MacBook-Pro-Damian web % vercel build --cwd ./
Vercel CLI 28.5.5
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Set up “~/Development/company/project/packages/website/web”? [Y/n] n

user@MacBook-Pro-Damian web % vercel build --cwd ../
Vercel CLI 28.5.5
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Set up “~/Development/company/project/packages”? [Y/n] 

user@MacBook-Pro-Damian web % vercel build --cwd ./../
Vercel CLI 28.5.5
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Set up “~/Development/company/project/packages”? [Y/n] 

user@MacBook-Pro-Damian web % vercel build --cwd .././
Vercel CLI 28.5.5
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Set up “~/Development/company/project/packages”? [Y/n] 

user@MacBook-Pro-Damian web % vercel build --cwd ../../
Vercel CLI 28.5.5
? No Project Settings found locally. Run `vercel pull` for retrieving them? [Y/n] y
? Set up “~/Development/company”? [Y/n] n
```

If I type `../../` I go back to 4 levels up. I think it shouldn't work that.

**Comment by amyegan** [comment]
Hi @dtokarczyk! It looks like you skipped the project setup steps in the prompt. You need to set up a new project, or link to an existing project, so `vercel build` can pull in the correct project settings. 

The [`vercel build`](https://vercel.com/docs/cli/build#) command should be used at the root of your repository after you [link](https://vercel.com/docs/cli/link#) the Vercel project to it. If you are using a [monorepo](https://vercel.com/blog/monorepos), then the build command should be run at the monorepo root. You can learn more about Vercel CLI at [vercel.com/docs/cli/deploying-from-cli](https://vercel.com/docs/cli/deploying-from-cli#)

**Comment by TooTallNate** [comment, marked as answer]
This was a bug. Nice catch. Fixed here https://github.com/vercel/vercel/pull/8949.

**Reply by dtokarczyk** [reply]
Thanks for fast reply! 🙏
